src/test/ui-test/install.test.ts

```diff
@@ -0,0 +1,242 @@
+import { expect } from 'chai';
+import * as fs from 'fs';
+import { repeat, TimeoutError } from '@theia-extension-tester/repeat';
+import {
+    EditorView,
+    error,
+    ExtensionsViewItem,
+    ExtensionsViewSection,
+    InputBox,
+    SideBarView,
+    ViewControl
+} from 'vscode-extension-tester';
+import { Workbench } from 'vscode-uitests-tooling';
+
+describe('Marketplace presentation', function () {
+    this.timeout(30000);
+    this.slow(10000);
+
+    let extensionMetadata: { [key: string]: any };
+    let name: string;
+    let marketplace: ExtensionsViewSection;
+
+    before('Load metadata', function () {
+        extensionMetadata = JSON.parse(fs.readFileSync('package.json', {
+            encoding: 'utf-8'
+        }));
+        name = extensionMetadata['displayName'];
+    });
+
+    beforeEach('Open marketplace', async function () {
+        marketplace = await openMarketplace(this.timeout() - 1000);
+    });
+
+    afterEach('Close marketplace', async function () {
+        await closeMarketplace(this.timeout() - 1000);
+    });
+
+    it('Has correct title', async function () {
+        const [_, extension] = await openExtensionPage(marketplace, name, this.timeout() - 1000);
+        expect(await extension.getTitle()).to.be.equal(extensionMetadata['displayName']);
+    });
+
+    it('Has correct description', async function () {
+        const [_, extension] = await openExtensionPage(marketplace, name, this.timeout() - 1000);
+        expect(await extension.getDescription()).to.be.equal(extensionMetadata['description']);
+    });
+
+    it('Has correct author', async function () {
+        const [_, extension] = await openExtensionPage(marketplace, name, this.timeout() - 1000);
+        expect(await extension.getAuthor()).to.be.equal('Red Hat');
+    });
+
+    it('Extension is installed', async function () {
+        const [_, extension] = await openExtensionPage(marketplace, name, this.timeout() - 1000);
+        expect(await extension.isInstalled()).to.be.true;
+    });
+
+    it('All commands are available', async function () {
```

Comment:
this test seems to not fit in the "Marketplace  presentation" test suite.

Reply:
True, I will move it somewhere else or rename suite if I find better name.